packages/web-components/src/components/va-radio/va-radio.scss

```diff
@@ -1,12 +1,24 @@
-@import '../../mixins/accessibility.css';
-@import '../../mixins/form-field-error.css';
+@use 'uswds-helpers/src/styles/usa-sr-only';
+@use 'usa-form/src/styles/usa-form';
+@use 'usa-fieldset/src/styles/usa-fieldset';
+@use 'usa-legend/src/styles/usa-legend';
+@use 'usa-label/src/styles/usa-label';
+@use 'usa-hint/src/styles/usa-hint';
+@use 'usa-error-message/src/styles/usa-error-message';
+
+@import '../../mixins/uswds-error-border.css';
```

Comment:
Unfortunately, it does not compile if you remove the final path value:

![Screen Shot 2022-12-01 at 4 09 32 PM](https://user-images.githubusercontent.com/872479/205169715-4cd306bd-e3f1-4df6-9a7f-9895fc96cae1.png)

Reply:
As for having a global scss file, I think that might make sense at some point but in this specific situation, we only want `usa-sr-only`. We don't necessarily want all of the `uswds-helpers` so I think it might make more sense to keep the import as it is right now where it's very specific about what it needs (instead of a whole bunch of stuff this component isn't using).

Reply:
Since the `scss` files are not in the same project you need to import them

Try:
```
@import 'usa-form/src/styles';
```

Reply:
You are welcome to experiment with different imports locally and let me know if there is a better way but we went through a discovery about how to import these modules earlier in the development process and the preferred way is with `use`. See this discussion with the Stencil developers:

https://stencil-worldwide.slack.com/archives/C79EANFL7/p1667581320118309

![Screen Shot 2022-12-02 at 8 54 52 AM](https://user-images.githubusercontent.com/872479/205320968-2ebbcb22-6655-46a9-8f41-758cd253ebb2.png)

Reply:
Sounds good!